source/app/web/statics/insights/index.html

```diff
@@ -314,7 +314,7 @@ <h2>
                 <p>
                   <b>{{ user }}</b> has used {{ languages.length }} different language{{ format("plural", languages.length) }} for a total of {{ format("number", languages.total) }} byte{{ format("plural", languages.total) }}.
                   <small class="footnote">
-                    Note that these numbers are based on results produced by <a href="github.com/github/linguist">GitHub linguist</a> from repositories owned by <b>{{ user }}</b> and that they may not be entirely accurate.
```

Comment:
Metrics uses LinguistJS now, should this be changed to link to that instead?

Reply:
Yeah we should update it 👍